src/Microsoft.AspNet.Hosting.Abstractions/EnvironmentName.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Hosting
+{
+    public static class EnvironmentName
+    {
+        public const string Development = "Development";
+        public const string Production = "Production";
```

Comment:
@davidfowl I made them static readonly before I made a PR, nothing but there's no possibility to change the values in the construct, so I think the constant will make sense